modules/service/src/main/resources/application.conf

```diff
@@ -42,4 +42,27 @@ akka {
     journal.plugin = ${app.persistence.journal-plugin}
     snapshot-store.plugin = ${app.persistence.snapshot-store-plugin}
   }
+
+ kafka.producer {
+
+    # Tuning parameter of how many sends that can run in parallel.
+    parallelism = 100
+    batch.size = 100
```

Comment:
do batches also have a max timeout? with the top level permissions usage pattern we don't expect too many writes... having this number set to a high value could increase the latency of event propagation between services.

Reply:
I only added this config, because I thought the `batch.size` was too big and the messages were not sent, but there was some other bug causing this.
After some reading my understanding is that this is an upper limit on how many messages Kafka will send if they come in quick succession, but shouldn't have impact on latency in case of small throughput.
> By default a buffer is available to send immediately even if there is additional unused space in the buffer. 


>The producer groups together any records that arrive in between request transmissions into a single batched request. Normally this occurs only under load when records arrive faster than they can be sent out.

Reply:
Leave the config in, it's quite useful.